src/components/List.js

```diff
@@ -51,6 +53,47 @@ const List = ({ firestore }) => {
     setInputText('');
   };
 
+  //* Created two lists
+  const inactiveList = [];
+  const activeList = [];
+
+  //* Used this function to divide active and inactive items
+  filteredList.map(item => {
+    let formatedLastPurchaseDate = dayjs.unix(
+      item.lastPurchasedDate['seconds'],
+    );
+    let formatedToday = dayjs();
+    let difference = formatedToday.diff(formatedLastPurchaseDate, 'd');
+    if (
+      item.numberOfPurchases <= 1 ||
+      parseInt(difference) >= item.nextPurchase * 2
+    ) {
+      inactiveList.push(item);
+    } else {
+      activeList.push(item);
+    }
+  });
```

Comment:
This is so readable! I don't think I've seen any of the cohorts I've worked with quite do it like this and I think it's a really interesting approach. Great work!